lib/utils/export/to-zip.js

```diff
@@ -0,0 +1,21 @@
+import JSZip from 'jszip';
+
+export const noteExportToZip = notes => {
+	const zip = new JSZip();
+
+	notes
+		.activeNotes
+		.forEach(
+			( { id, content } ) => zip.file( `${ id }.txt`, content )
```

Comment:
> Also users have requested that the tags be in the exported file as well. Maybe just at the bottom of the file?

Added list of tags to end of note in 9d11ff0.

@roundhill in this iteration I have explicitly declared the list of tags at the end with a header because it was a bit confusing to me in my tests just having the space-separated list of hashed-tags.

e.g.
```
Chocolate Cake

Take the eggs and beat them with sugar.
Pour in chocolate.
Bake.

Tags:
 - #cake
 - #chocolate
 - #kurz-gesagt
 - #recipes
```